indexer/services/comlink/src/controllers/api/v4/asset-positions-controller.ts

```diff
@@ -52,8 +52,8 @@ const router: express.Router = express.Router();
 const controllerName: string = 'asset-positions-controller';
 
 @Route('assetPositions')
-class AddressesController extends Controller {
```

Comment:
This is a breaking change for the API, it should be created as a separate API and not replacing the current API.